hack/quick-install.sh

```diff
@@ -32,8 +32,7 @@ delete() {
   helm delete karpenter || true
   helm delete cert-manager --namespace cert-manager || true
   helm delete kube-prometheus-stack --namespace monitoring || true
-
-  kubectl delete namespace cert-manager monitoring || true
+  kubectl delete namespace karpenter cert-manager monitoring || true
```

Comment:
Shall we include clean up of the resources created by cloud formation?

Reply:
These details are in the AWS cleanup script.